docs/api/cases/cases-api-update.asciidoc

```diff
@@ -0,0 +1,235 @@
+[[cases-api-update]]
+== Update cases API
+++++
+<titleabbrev>Update cases</titleabbrev>
+++++
+
+Updates one or more cases.
+
+=== Request
+
+`PATCH <kibana host>:<port>/api/cases`
+
+=== Prerequisite
+
+You must have `all` privileges for the *Cases* feature in the *Management*,
+*{observability}*, or *Security* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>.
+
+=== Request body
+
+`cases`::
+(Required, array of objects) Array containing one or more case objects.
++
+.Properties of `cases` objects
+[%collapsible%open]
+====
+`connector`::
+(Optional, object) An object that contains the connector configuration.
++
+.Properties of `connector`
+[%collapsible%open]
+=====
+`fields`::
+(Required, object) An object containing the connector fields.
++
+--
+To remove the connector, specify `null`. If you want to omit any individual
+field, specify `null` as its value.
+
+For {sn-itsm} connectors, refer to <<servicenow-action-configuration>>. For
+example:
+
+* `urgency`: The urgency of the incident.
+* `severity`: The severity of the incident.
+* `impact`: The impact of the incident.
```

Comment:
Could you reuse the description from the `create` docs here, which is slightly more informative?